src/vendors.js

```diff
@@ -55,5 +55,5 @@ export const vendored = {
   reselect,
   // Svg is required via relative paths, so the key is not imported path.
   // See .babel/transform-mc.js
-  Svg
+  BreakpointSvg
```

Comment:
I don't know if I did it correctly. I recursively checked devtools-core package and saw that no one uses it. devtools-core uses it's own version of `Svg.js`